velox/docs/functions/spark/binary.rst

```diff
@@ -4,19 +4,32 @@ Binary Functions
 
 .. spark:function:: hash(x, ...) -> integer
 
-    Computes the hash of one or more input values using seed value of 42.
+    Computes the hash of one or more input values using seed value of 42. Each
+    input's type can be different from each other's.
```

Comment:
nit: What does 'Each input's type can be different from each other's' mean ?

Reply:
This function can have multiple inputs, and their types can be different.

Reply:
Looks easily misleading. Just modified. Thanks! cc @mbasmanova